releasenotes/notes/apm-obfuscation-sql-query-8db35fd42138d57b.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    The `sql.query` tag will now always be obfuscated by the Datadog Agent
+    even if this tag was already set by a tracer or manually by a customer.
```

Comment:
Just a question, but do we refer to the "customer" in release notes? I'm not sure, but it just stood out to me because if I imagine I am a customer/user reading this, it'd be a little odd.

For example, I would imagine we'd instead say "...even if a tracer already sets this tag or you set the tag manually."

Reply:
Agreed with @brett0000FF and added suggestions accordingly.